syntax = "proto3";

import "shiva/core/communication_objects/env_specs.proto";
import "shiva/core/communication_objects/env_step.proto";
import "shiva/core/communication_objects/helpers.proto";

package communication_objects;

service Environment {
    rpc GetSpecs (Empty) returns (EnvironmentSpecs) {}
    rpc Step (EnvStepInput) returns (EnvStepOutput) {}
    rpc Reset (Empty) returns (EnvStepOutput) {}
}